avaframe/runOperational.py

```diff
@@ -0,0 +1,92 @@
+"""
+    Run script for running the operational workflow
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import time
+import logging
+
+# Local imports
+from avaframe.com1DFA import com1DFA
+from avaframe.log2Report import generateReport as gR
+from avaframe.out3SimpPlot import outPlotAllPeak as oP
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+from avaframe.in3Utils import initializeProject as initProj
+
+# Time the whole shebang
+startTime = time.time()
+
+# log file name; leave empty to use default runLog.log
+logName = 'runOperational'
```

Comment:
I can't find the log that is created... is it supposed to be in the ava directory?

Reply:
Ah, yes, this needs to be caught: it is currently deleted in the clean script ...